README.md

```diff
@@ -1 +1,1146 @@
-# eip-message-transformation
\ No newline at end of file
+[![Build Status](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)
+
+# eip-message-transformation
+There are different ways of message transformation methods in EIP (Enterprise Integration Patterns). In this guide, we are focusing on *content filter*, *claim check* and *content enricher* message transformation methods between services using an example scenario.
+
+> This guide describes implementing three message transformation patterns using Ballerina programming language as simple steps.
```

Comment:
Might be better to say "in simple steps"

Reply:
Done